src/Service/S3/src/Enum/Event.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AsyncAws\S3\Enum;
+
+final class Event
+{
+    public const S_3_OBJECT_CREATED_ = 's3:ObjectCreated:*';
```

Comment:
Oh... I don't like the name of this constant.
I'll fix something in the generator.

Reply:
https://github.com/async-aws/aws/pull/865

Reply:
Nice. @sunkan, could you rebase and rerun the generator please?